chart/jenkins-operator/templates/jenkins.yaml

```diff
@@ -85,6 +85,9 @@ spec:
   {{- with .Values.jenkins.notifications }}
   notifications: {{ toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .Values.jenkins.nodeSelector }}
+  nodeSelector: {{ toYaml . | nindent 6 }}
```

Comment:
```suggestion
  nodeSelector: {{ toYaml . | nindent 4 }}
```

Reply:
Hi @prryb , thanks for the noticed, changed in commit 4b75d27bc0f92c1a7f22a751ac43df4ebe5f9fe3